### ISSUE TITLE
Curious Question:  does and image with role="presentation" AND alt="blah blah blah" fail WCAG 1.1.1?

### ISSUE THREAD

**Issue by goodwitch** [dequelabs/axe-core#3268]
**Product:** axe DevTools extension

**Expectation:** I'm thinkin' that some people may think it is worth throwing a WCAG 1.1.1 failure in the following situation (an image has role="presentation" AND has alternative text that is not null).  For example:

`<img alt="Country Image" role="presentation"  src="/countryFlagImages/OvalCopy4.png" >`

**Actual:**  I created a test page at http://www.glendathegood.com/a11y/alt/presentation.html and found that the axe DevTools extension does not call a WCAG 1.1.1 failure on this code. 

**Motivation:** In the WAI tutorial on Decorative Images at https://www.w3.org/WAI/tutorials/images/decorative/ there is a sentence that reads "Screen readers also allow the use of WAI-ARIA to hide elements by using role="presentation". However, currently, this feature is not as widely supported as using a null alt attribute."  This leaves me wondering...should this be a failure of 1.1.1 (or does someone need to update the tutorial because most screen readers now really do respect role="presentation".

Also...seems like a conflict.  Like, make up your mind.  Seems to me like an image with role of presentation should not have alt text. 

<code><pre>
axe-core version: 4.3.5
axe DevTools extension: v4.18.2
Browser and Assistive Technology versions: Chrome 95
Platform:  Mac OS Big Sur 11.6
</code></pre>

**Comment by straker** [comment]
Interesting. The code should throw an error for `aria-allowed-role` (which is a best practice rule), is that sufficient or should this specifically be a WCAG 1.1.1 failure?

> "Fix any of the following:
>  ARIA role presentation is not allowed for given element"

**Comment by goodwitch** [comment]
Doh!  Thanks for reminding me to look at BPs (I had them turned off).  That helps me come to terms with this.  And for extra credit...I'm testing to see how the main screen readers are handling this today.  So far...so good.  I'm adding a table at the bottom of this page (and filling it in as I test). http://www.glendathegood.com/a11y/alt/presentation.html

**Comment by WilcoFiers** [comment]
Think we don't need this. Closing it.
